docs/2023/QuML/Enhanced quml player/updates/2023-07-21.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Week 3
+author: Yashi Gupta
+---
+
+## Milestones
+- [x] Build logic behind different question types - Milestone 1
+- [x] Making new question types - Milestone 2
+- [x] Documentation of code - Milestone 3
+- [x] Logical impelemtation of new question types - Milestone 4
+- [x] Raising PR - Milestone 5
+
+## Screenshots / Videos 
+![Alt text](<../../../../../images/Screenshot 2023-07-25 004536.png>)
+![Alt text](<../../../../../images/Screenshot 2023-07-25 004808.png>)
+![Alt text](<../../../../../images/Screenshot 2023-07-25 005329.png>)
+![Alt text](<../../../../../images/Screenshot 2023-07-25 005533.png>)
+
+## Contributions
+-[ Successfully build the logic behind the questions]
+-[ Successfully made the UI of new question types]
+-[ Successfully completed the documentation of code]
+-[ working on logic behind new question types]
+-[ Raised PR] 
```

Comment:
Attach Link of PRs in the contribution section or leave it empty if no PR. For all the weeks

Reply:
I have made the changes , and raise PR from different branch,

Reply:
Can we close this PR then?